java/build.gradle

```diff
@@ -20,7 +20,6 @@ dependencies {
 sourceSets {
     test {
         java {
-            srcDir 'example'
```

Comment:
Is there a separate way that we could add the examples into this build file as optional? libsbp's current docker doesn't include mvn. Or maybe there would be value in including a separate gradle file just for building the examples?

Reply:
hmmmmm could swap it to use gradle instead of maven but wanted to just write a sample outline project of how to implement it, and could also be runnable by using `mvn install` if needed.

but switching entirely to gradle would be fine as well, was thinking like an example project not within the current `build.gradle`, doesn't matter on whether its maven or gradle - (not necessary in building it with docker, just for demo for how to implement it + a runnable demo if needed)

Reply:
I think a separate Gradle sounds reasonable and could add a step to the Makefile to make java-examples or something as well.